### ISSUE TITLE
Go to definition with relative imports

### ISSUE THREAD

**Issue by DonJayamanne** [davidhalter/jedi#1029]
Version - Master branch of Jedi
Go to definition of `print_greeting` in `test.py` does not work.

Project structure:
```
-stopwatch_app
--test.py
--utils
---greeter.py
```

And contents of:

```
# test.py

from utils.greeter import print_greeting

def main():
    print_greeting()
```

```
# utils/greeter.py

def print_greeting():
   print('Hello')
```

[stopwatch_app.zip](https://github.com/Microsoft/vscode-python/files/1493474/stopwatch_app.zip)

**Comment by imdadahad** [comment]
Any update?

**Comment by davidhalter** [comment]
It's been only 27 days. You seem to be having a lot of expectations ;)

However, I'm actually woring on imports at the moment. Not exactly on this problem, but it might fix this issue as well.

**Comment by davidhalter** [comment]
Ok. This issue is kind of invalid. It works on Python3 if I fix the import in `test.py`. Also there are no relative imports. Everything here is about absolute imports.

However it involves implicit namespace packages where some improvements have just been merged (#959) and a fix to relative imports (#973).

Let me know if you have further problems.
